backend/datasource.ts

```diff
@@ -0,0 +1,16 @@
+export const dataSource = {
+  type: 'mysql',
+  host: 'localhost', 
+  port: 3306, 
+  username: 'root', 
+  password: 'your_password', 
+  database: 'your_database',
```

Comment:
I think this is where docker will play its role

Reply:
agreed